benchmarks/experiment_runner.py

```diff
@@ -193,8 +205,8 @@ def generate_and_run_all_configs(self):
                              benchmark_model.to_dict(), {"error": str(e)})
 
   # TODO: Use `_unique_basename` instead.
-  def _get_config_fingerprint(self, experiment_config: OrderedDict,
-                              model_config: OrderedDict) -> str:
+  def _get_config_fingerprint(self, experiment_config: OrderedDict[str, Any],
+                              model_config: OrderedDict[str, Any]) -> str:
```

Comment:
Can we narrow `Any` with `Optional[str]`?

Reply:
Same for the other `OrderedDict[str, Any]` occurrences.

Reply:
It can be mapped to bool also. Basically the value can be None, bool, str.

Reply:
Updated, check same comments above.